addons/l10n_id_efaktur_coretax/views/account_move.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_move_efaktur_form_view" model="ir.ui.view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="l10n_id_efaktur.account_move_efaktur_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='l10n_id_tax_number']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='l10n_id_kode_transaksi']" position="attributes">
+                <attribute name="attrs">{'invisible': [('l10n_id_efaktur_available', '=', False)], 'required': [('l10n_id_efaktur_available', '=', True)]}</attribute>
+            </xpath>
+            <xpath expr="//field[@name='l10n_id_replace_invoice_id']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
```

Comment:
No longer required?

Reply:
yes seems like it,

what the accounting team in ID said is that people just re-submit the XML